src/catalog/src/builtin.rs

```diff
@@ -6689,14 +6689,23 @@ ON mz_internal.mz_recent_activity_log_thinned (sql_hash)",
 };
 
 pub const MZ_KAFKA_SOURCES_IND: BuiltinIndex = BuiltinIndex {
-    name: "mz_kafka_sources",
+    name: "mz_kafka_sources_ind",
```

Comment:
I've implemented this change in https://github.com/MaterializeInc/materialize/pull/26854, so you could remove it from this PR since looks like you gotta go through another test round anyway.